package.json

```diff
@@ -17,7 +17,7 @@
     "test:workspace": "esno ./scripts/index.ts test -w -v",
     "test": "esno ./scripts/index.ts test -v",
     "test:dev": "vitest --mode=development",
-    "test:prod": "PROD=1 DEV= vitest",
+    "test:prod": "PROD=1 DEV= pnpm -r test:specs",
```

Comment:
@wycats I made this change at the recommendation of @cafreeman, because running vitest from the root via `pnpm vitest` doesn't appear to pick up the local vite config *or* local dependencies for each package. This new script will execute the tests within each package with the correct env variables set.